public/docs/_examples/pipes/dart/lib/fetch_json_pipe.dart

```diff
@@ -8,12 +8,12 @@ import 'package:angular2/angular2.dart';
 // #docregion pipe-metadata
 @Pipe(name: 'fetch', pure: false)
 // #enddocregion pipe-metadata
-@Injectable() // FIXME(chalin): unnecessary?
 class FetchJsonPipe extends PipeTransform {
   dynamic _fetchedValue;
   Future<dynamic> _fetchPromise;
 
-  transform(String url) {
+  // beta.15+ transform(String url) {
```

Comment:
Please remove, we can easily find this code back if needed.

Link to Dart beta.16 update: https://github.com/angular/angular.io/commit/0557c728d75e519bc6846b37492414dff540c6ee